models/r1_rover/r1_rover.sdf.jinja

```diff
@@ -570,7 +570,7 @@
       <sdk_udp_port>14540</sdk_udp_port>
       <hil_mode>false</hil_mode>
       <hil_state_level>false</hil_state_level>
-      <enable_lockstep>true</enable_lockstep>
+      <enable_lockstep>false</enable_lockstep>
```

Comment:
Is there a reason to disable lockstep?

Reply:
Ah.. that may be my mistake. As I remember, that wouldn't do effect on the sim stability.